### DISCUSSION TITLE
Add a 2nd time to select a timezone

### DISCUSSION THREAD

**Discussion by Whydidthechickencrosstheroad** [monkeytypegame/monkeytype#5814] [Ideas] [answered]
What I mean is that if you accidentally chose the wrong timezone, there is a option that you can change it (you can change it every 30 days) and I accidentally chose the wrong timezone before the local timezone I live in feature was added. It is going to be at the account settings area and it will give you a time (by days, hours, minutes, and seconds) and you can change only once if you change the timezone. If you change the timezone, only the time streak time will select the time when midnight at that time and nothing else is impacted. If they chose accidentally the wrong option, they will show a popup question to confirm (if it is not matched the timezone for your local time)

**Comment by Miodec** [comment]
For mistakes like these they dont happen often so i just do it manually. Feel free to email me at `support@monkeytype.com`

**Reply by Whydidthechickencrosstheroad** [reply]
Can you change my timezone by the way?

**Reply by Miodec** [reply, marked as answer]
Yes, just email me.

**Reply by Whydidthechickencrosstheroad** [reply]
Ok
